cabal.project

```diff
@@ -157,6 +157,13 @@ source-repository-package
     subdir: generated
 
 --------------------------------------------------------------------------------
+
+source-repository-package
+    type: git
+    location: https://github.com/paolino/int-cast.git
+    tag: 0232b037552a4247b4a7ac21f816cc77cc95f7a4
+    --sha256: 1pmghd9pz35ycxp0c3f5sz4fghfi6ywjmp6wm9n45f8l5ah9vhhw
```

Comment:
I've just made a PR to upgrade `int-cast` to support GHC versions 9.8 and 9.10 here:
- https://github.com/haskell-hvr/int-cast/pull/9